Yafc/Widgets/MainScreenTabBar.cs

```diff
@@ -65,13 +66,19 @@ private void BuildContents(ImGui gui) {
                     gui.DrawRectangle(new Rect(gui.lastRect.X, gui.lastRect.Bottom - 0.4f, gui.lastRect.Width, 0.4f), isActive ? SchemeColor.Primary : SchemeColor.Secondary);
                 }
 
-                var evt = gui.BuildButton(gui.lastRect, isActive ? SchemeColor.Background : SchemeColor.BackgroundAlt, (isActive || isSecondary) ? SchemeColor.Background : SchemeColor.Grey);
+                var evt = gui.BuildButton(gui.lastRect, isActive ? SchemeColor.Background : SchemeColor.BackgroundAlt, (isActive || isSecondary) ? SchemeColor.Background : SchemeColor.Grey, button: 0);
                 if (evt == ButtonEvent.Click) {
-                    changePage = InputSystem.Instance.control ? 2 : 1;
-                    changePageTo = page;
+                    if (gui.actionParameter == SDL.SDL_BUTTON_RIGHT) {
+                        gui.window?.ShowTooltip(null); // hide the tab tooltip, otherwise it's displayed over the dropdown
```

Comment:
What do you think about making a separate function `HideTooltip` and using it instead of passing a `null` here?
Something like
```
public void HideTooltip() {
    this.tooltip = null;
    Rebuild();
}
```

My main concern is readability. Without this use case, it's not as obvious why `ShowTooltip()` accepts null.
On the other hand, there might be things that I missed, so feel free to correct me.

Reply:
Good idea! I have changed that as you suggest